### PR TITLE
fixing issue 60 re InDesign Server issues exporting a PDF

### PR DESCRIPTION
**Pull request by kylehodgson**

### CODE REVIEW

File: src/simple_idml/indesign/scripts/export.jsx [hunk 1]
```diff
@@ -204,24 +210,41 @@ try {
...
                 printerMarkWeight = PDFMarkWeight.p125pt;
                 registrationMarks = _registrationMarks;
                 try {
                     simulateOverprint = false;
                 }
-                catch(e){}
+                catch(e) { alert("ignoring exception setting property simulateOverprint: " + e)}
                 //Set viewPDF to true to open the PDF in Acrobat or Adobe Reader.
-                viewPDF = false;
+                //viewPDF = false;
```
Comment (Starou) [inline src/simple_idml/indesign/scripts/export.jsx:228]: Is it wanted?

File: src/simple_idml/indesign/scripts/export.jsx [hunk 2]
```diff
@@ -144,7 +144,7 @@ try {
                 try{
                     ignoreSpreadOverrides = false;
                 }
-                catch(e){}
+                catch(e){ alert("Ignoring exception setting property ignoreSpreadOverrides: " + e)}
```
Comment (Starou) [inline src/simple_idml/indesign/scripts/export.jsx:147]: For the various messages, what do you think of something like : 
`alert('Warning: cannot set "ignoreSpreadOverrides" option. Error was: ' + e)` ?

File: src/simple_idml/indesign/scripts/export.jsx (hunk 1 above)
Comment (kylehodgson) [reply src/simple_idml/indesign/scripts/export.jsx:228]: Ah, good catch. I'll convert that to a try/catch block like the others.

File: src/simple_idml/indesign/scripts/export.jsx (hunk 2 above)
Comment (kylehodgson) [reply src/simple_idml/indesign/scripts/export.jsx:147]: Yep, I like that. Will adjust.

**Comment by kylehodgson** [comment]
ok, fixed both of those in a new commit. thanks for the quick response!

**Comment by Starou** [comment]
Sounds good to me!
Can you squash the 2 commits please? There is also an additional tabulation/spaces line 133 to remove.
I'll test the PR tomorrow.

**Comment by kylehodgson** [comment]
done.

**Comment by Starou** [comment]
Just release SimpleIDML-1.1.3. Thanks for the contribution @kylehodgson.

**Pull request merged**